tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/repopulate_person_post_deid_test.py

```diff
@@ -1,161 +1,174 @@
-        gender_nonbinary_concept_id = 1585841
-        gender_nonbinary_source_concept_id = 123
-        sex_female_concept_id = 1585847
-        sex_female_source_concept_id = 45878463
-        for tmpl in INSERT_FAKE_PARTICIPANTS_TMPLS:
-            query = tmpl.render(
-                project_id=self.project_id,
-                dataset_id=self.dataset_id,
-                gender_concept_id=GENDER_CONCEPT_ID,
-                gender_nonbinary_concept_id=gender_nonbinary_concept_id,
-                gender_nonbinary_source_concept_id=
-                gender_nonbinary_source_concept_id,
-                sex_at_birth_concept_id=SEX_AT_BIRTH_CONCEPT_ID,
-                sex_female_concept_id=sex_female_concept_id,
-                sex_female_source_concept_id=sex_female_source_concept_id)
-            try:
-                resp = bq_utils.query(query)
-            except HttpError as e:
-                self.fail("failed to execute query '{}': {}".format(
-                    query, e.content))
-            self.assertTrue(resp["jobComplete"])
-
-        clean_cdr_engine.clean_dataset(self.project_id, self.dataset_id,
-                                       self.sandbox_dataset_id,
-                                       [(RepopulatePersonPostDeid,)])
-
-        rows = bq_utils.response2rows(
-            bq_utils.query("SELECT * FROM `{}.{}.person`".format(
-                self.project_id, self.dataset_id)))
-        self.assertEquals(len(rows), 2)
-
-        by_participant = {r["person_id"]: r for r in rows}
-        self.assertPersonFields(
-            by_participant[1], {
-                "gender_concept_id": gender_nonbinary_concept_id,
-                "gender_source_value": "nonbinary_src",
-                "gender_source_concept_id": gender_nonbinary_source_concept_id,
-                "sex_at_birth_concept_id": sex_female_concept_id,
-                "sex_at_birth_source_value": "female_src",
-                "sex_at_birth_source_concept_id": sex_female_source_concept_id
-            })
-        self.assertPersonFields(
-            by_participant[2], {
-                "gender_concept_id": 0,
-                "gender_source_value": "No matching concept",
-                "gender_source_concept_id": 0,
-                "sex_at_birth_concept_id": 0,
-                "sex_at_birth_source_value": "No matching concept",
-                "sex_at_birth_source_concept_id": 0
-            })
+        fq_dataset_name = self.fq_table_names[0].split('.')
+        self.fq_dataset_name = '.'.join(fq_dataset_name[:-1])
+
+        self.date = parser.parse('2020-05-05').date()
+
+        super().setUp()
+
+    def test_repopulate_person_post_deid(self):
+        """
+        Tests that the specifications for queries perform as designed.
+
+        Validates pre conditions, tests execution, and post conditions based on the load
+        statements and the tables_and_counts variable.        
+        """
+
+        queries = []
```

Comment:
nit: this line can be removed because of `queries` on line 147

Reply:
Sounds good, just pushed